Hudl.Mjolnir/Breaker/FailurePercentageCircuitBreakerConfig.cs

```diff
@@ -1,51 +1,60 @@
-﻿using Hudl.Mjolnir.External;
-using Hudl.Mjolnir.Key;
+﻿using Hudl.Mjolnir.Key;
 using System;
+using Hudl.Mjolnir.Config;
 
 namespace Hudl.Mjolnir.Breaker
 {
     internal class FailurePercentageCircuitBreakerConfig : IFailurePercentageCircuitBreakerConfig
     {
-        private readonly IMjolnirConfig _config;
+        private readonly MjolnirConfiguration _config;
 
-        public FailurePercentageCircuitBreakerConfig(IMjolnirConfig config)
+        public FailurePercentageCircuitBreakerConfig(MjolnirConfiguration config)
         {
             _config = config ?? throw new ArgumentNullException(nameof(config));
         }
 
+        private BreakerConfiguration _getBreakerConfiguration(GroupKey key)
+        {
+            BreakerConfiguration breakerConfiguration;
+    
+            return _config.BreakerConfigurations.TryGetValue(key.Name, out breakerConfiguration) ? 
+                breakerConfiguration : 
+                _config.DefaultBreakerConfiguration;
```

Comment:
Feels odd to have the caller know that it needs to fall back to the default. Can this behavior be put into the config class instead? This is related to my other comment on `MjolnirConfiguration` about encapsulating the `Dictionary`.